packages/flutter/lib/src/material/button_style.dart

```diff
@@ -435,3 +435,19 @@ class _LerpShapes implements MaterialStateProperty<OutlinedBorder?> {
     return ShapeBorder.lerp(resolvedA, resolvedB, t) as OutlinedBorder?;
   }
 }
+
+/// How the icon in any `button.icon` should be placed, where `button` could be:
+/// * [TextButton]
```

Comment:
you need a blank line before a bulleted list in our markdown syntax.

Reply:
Sure, I will add it 🚀.